docs/reference/cat/thread_pool.asciidoc

```diff
@@ -2,106 +2,124 @@
 
 [source,sh]
 --------------------------------------------------
 % curl 192.168.56.10:9200/_cat/thread_pool
-host1 192.168.1.35 0 0 0 0 0 0 0 0 0
-host2 192.168.1.36 0 0 0 0 0 0 0 0 0
+0EWUhXe bulk                0 0 0
+0EWUhXe fetch_shard_started 0 0 0
+0EWUhXe fetch_shard_store   0 0 0
+0EWUhXe flush               0 0 0
+0EWUhXe force_merge         0 0 0
+0EWUhXe generic             0 0 0
+0EWUhXe get                 0 0 0
+0EWUhXe index               0 0 0
+0EWUhXe listener            0 0 0
+0EWUhXe management          1 0 0
+0EWUhXe refresh             0 0 0
+0EWUhXe search              0 0 0
+0EWUhXe snapshot            0 0 0
+0EWUhXe warmer              0 0 0
 --------------------------------------------------
 
-The first two columns contain the host and ip of a node.
+The first column is the node name
 
 [source,sh]
 --------------------------------------------------
-host      ip
-host1 192.168.1.35
-host2 192.168.1.36
+node_name
+0EWUhXe
 --------------------------------------------------
 
-The next three columns show the active queue and rejected statistics for the bulk thread pool.
-
+The second column is the thread pool name
 [source,sh]
 --------------------------------------------------
-bulk.active bulk.queue bulk.rejected
-          0          0             0
+name
+bulk
+fetch_shard_started
+fetch_shard_store
+flush
+force_merge
+generic
+get
+index
+listener
+management
+refresh
+search
+snapshot
+warmer
 --------------------------------------------------
 
-The remaining columns show the active queue and rejected statistics of the index and search thread pool respectively.
 
-Also other statistics of different thread pools can be retrieved by using the `h` (header) parameter.
+The next three columns show the active, queue, and rejected statistics for each thread pool
 
 [source,sh]
 --------------------------------------------------
-% curl 'localhost:9200/_cat/thread_pool?v&h=id,host,suggest.active,suggest.rejected,suggest.completed'
-host      suggest.active suggest.rejected suggest.completed
-host1                  0                0                 0
-host2                  0                0                 0
+active queue rejected
+     0     0        0
+     0     0        0
+     0     0        0
+     0     0        0
+     0     0        0
+     0     0        0
+     0     0        0
+     0     0        0
+     0     0        0
+     1     0        0
+     0     0        0
+     0     0        0
+     0     0        0
+     0     0        0
 --------------------------------------------------
 
-Here the host columns and the active, rejected and completed suggest thread pool statistic are displayed. The suggest
-thread pool won't be displayed by default, so you always need to be specific about what statistic you want to display.
-
-[float]
-=== Available Thread Pools
+The cat thread pool API accepts a `thread_pools` URL parameter for specifying regular expressions to
```

Comment:
You added support for `/_cat/thread_pools/search,generic`, do you want to mention that too?